app/assets/stylesheets/_github-ribbon.css.sass

```diff
@@ -0,0 +1,48 @@
+// Responsive GitHub Ribbon
+
+.ribbon
+  background-color: #7d7d7d
+  overflow: hidden
```

Comment:
The only problem I seem to have is that a horizontal scrollbar comes up, even though I put `overflow: hidden`. Even when I put: `overflow: hidden !important`, or when I repeat it on the parent element (header), it doesn't seem to work :( ideas? @alicetragedy @shannonethomas @gerwitz?